### ISSUE TITLE
ERROR opentty: open /dev/pts/0: permission denied

### ISSUE THREAD

**Issue by jdx** [theimpostor/osc#7]
not sure what this error means, I am probably doing something wrong. System is Ubuntu 24.04.

```
jdx in 🌐 bamboo in ~ ❯ osc copy
2024-05-11T14:55:40Z ERROR opentty: open /dev/pts/0: permission denied
Error: open /dev/pts/0: permission denied
Usage:
  osc copy [flags]

Flags:
  -h, --help   help for copy

Global Flags:
  -d, --device string   select device (default "/dev/pts/0")
  -l, --log string      write logs to file
  -t, --timeout float   tty read timeout in seconds (default 5)
  -v, --verbose         verbose logging
```

**Comment by theimpostor** [comment]
Can you try with unsetting SSH_TTY `SSH_TTY= osc copy`, or use the `--device` arg: `osc copy --device /dev/tty`

**Comment by ryuheechul** [comment]
I just experienced the same issue and I also believe I stumbled upon the report that talks about the root cause of this issue and it's described here: https://github.com/mobile-shell/mosh/issues/1134.

tl;dr: mosh leaves (no longer existing) `SSH_TTY` that mosh used for initial connection via SSH but then, that SSH_TTY (e.g. `/dev/pts/0`) no longer exists and the remote (mosh) shell is using a different tty, (say `/dev/pts/3`), hence the mismatch and the error.

As described in the linked issue, hope the mosh side tackles it and anyone who find this issue here may understand the reason behind this issue quicker.
